server.js

```diff
@@ -15,16 +14,14 @@ const router = express();
 // register view engine
 app.set("view engine", "ejs");
 
-let ngrokUrl = null;
 let callbackState = {};
 
 // Launch the Express server
 (async function bootstrap() {
   // Load environment variables
   dotenv.config();
   const port = process.env.PORT || "3000";
-  ngrokUrl = await ngrok.connect(port);
```

Comment:
Don't we need to keep ngrok for local testing?

Reply:
There are some issues when trying to connect to ngrok through the app. During local testing, we can run the ngrok separately and configure the callback url in the .env file. I'll update the readme with the needed tests.